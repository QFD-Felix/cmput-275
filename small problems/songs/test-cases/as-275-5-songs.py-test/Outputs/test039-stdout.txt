19 12 2 4 7 5 0 9 13 0 13 4 0 1 13 3 6 0 2 22 4 10 4 2 5 7 4 1 16 9 4 2 28 4 1 8 0 2 8 11 0 1 10 4 14 0 3 4 0 12 9 17 18 0 0 0 0 0 9 3 5 16 39 0 0 10 0 11 3 0 2 3 8 0 17 1 3 8 4 2 6 3 1 13 18 2 4 1 1 3 15 2 3 1 14 4 7 0 0 1 0 11 8 3 0 19 8 0 1 8 1 8 3 0 2 3 9 5 2 4 1 6 0 5 2 7 0 2 0 2 0 0 0 0
