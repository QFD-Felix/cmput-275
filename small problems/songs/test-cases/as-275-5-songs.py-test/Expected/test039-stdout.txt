19 12 2 4 7 3 0 9 13 0 0 4 0 1 13 3 6 0 2 22 4 10 4 0 1 7 3 1 16 9 0 2 28 4 0 0 0 2 8 11 0 1 10 0 3 0 3 4 0 9 9 17 18 0 0 0 0 0 9 2 5 16 0 0 0 10 0 11 3 0 2 3 8 0 17 1 3 8 4 2 6 3 1 13 18 2 4 1 1 3 15 2 3 1 14 4 7 0 0 1 1 11 8 3 0 19 8 0 1 8 1 8 3 0 2 3 9 5 2 4 1 8 6 7 2 9 1 4 0 0 0 2 0 0 0 0
